### DISCUSSION TITLE
Let's go

### DISCUSSION THREAD

**Discussion by kMutagene** [fslaborg/Graphoscope#1] [Discussions]
Hi @fslaborg/graphoscope,

Thanks for the great meeting today. I just found out that we can use Discussion boards on private repos, so I'd suggest we keep the repo private until we came up with some barebones implementations.

Just FYI, I set up a section for library design discussions and polls (this post is the first in the section).

**Comment by kMutagene** [comment]
@HarryMcCarney , can you invite Dogan (or post his GitHub handle so I can invite him)?

**Reply by HarryMcCarney** [reply]
Hi Kevin,

Yes it's 
DoganCK

Thanks,
Harry

**Reply by HarryMcCarney** [reply]
Have added him
